### DISCUSSION TITLE
How I can print my pictures using the command line

### DISCUSSION THREAD

**Discussion by xmha97** [ImageMagick/ImageMagick#4394] [Development]
Hi, how I can print a picture using the command line?

![image](https://user-images.githubusercontent.com/23485114/138111968-1edf6e3b-6b65-44fc-837d-08a8bf253b9e.png)

I need a parameter like this:
```
imdisplay.exe -print d:\default.jpg
```

**Comment by urban-warrior** [comment]
Try this command:

```
rundll32 C:\WINDOWS\system32\shimgvw.dll,ImageView_PrintTo "c:\mydir\my.jpg" "My HP Printer"
```
